commcare_connect/organization/decorators.py

```diff
@@ -18,6 +18,12 @@ def _request_user_is_admin(request):
     ) or request.user.is_superuser
 
 
+def _request_org_is_program_manager(request):
```

Comment:
Shouldn't this be`_request_user_is_program_manager`?

Reply:
Done [84a04d7](https://github.com/dimagi/commcare-connect/pull/379/commits/84a04d76f25496165c9c7b6a5085bc87ccc5284a)